specifyweb/frontend/js_src/lib/components/FormSliders/Slider.tsx

```diff
@@ -45,7 +45,7 @@ export function Slider({
       >
         {icons.chevronLeft}
       </Button.Small>
-      <div className="grid grid-cols-[1fr_auto_1fr] items-center gap-1 font-bold">
+      <div className="grid min-w-[theme(spacing.20)] grid-cols-[1fr_auto_1fr] items-center gap-1 font-bold">
```

Comment:
hmm, the slider is not supposed to have its width change

it has `grid-cols-[1fr_auto_1fr]` - meaning the number of the left and number on the right must be same width.

does that not work?

Reply:
if you look at the video in the original issue (#3784) you will see that the width change is likely caused by the textarea value being longer or shorter - if that's indeed the case, there are styles that can be applied that would forbid textarea to grow it's parent

BUT, fixing it for just text areas won't help - this bug can happen for many different reasons, and the reason depends on the form layout and whether record has subforms - a more generic solution is required - that is why @CarolineDenis you worked on the feature that was going to forbid dialogs from shrinking after expanding (to help minimize the layout shifts).

another solution you did was making dialogs remember their size when resized - if you manually resize the dialog, the bug in the original issue (#3784) likely won't happen

another generic solution is introducing keyboard shortcuts (see https://github.com/specify/specify7/issues/1746), which would reduce the need for reaching for the record slider at the top

Reply:
> hmm, the slider is not supposed to have its width change
> 
> it has `grid-cols-[1fr_auto_1fr]` - meaning the number of the left and number on the right must be same width.
> 
> does that not work?

Without min-w-[theme(spacing.20)]: 

https://github.com/specify/specify7/assets/108160931/23dff548-7af4-4bbf-8cdc-059d8c0f8de9

With it: 

https://github.com/specify/specify7/assets/108160931/e8ca9468-2da3-412a-adf9-a11a80170723

There is 2 shiftings, one in the slider and one in the full container.

Reply:
I am sorry to say, but this is not a solution, but only an ugly hack
`min-w-[theme(spacing.20)]`

Turns out it was just a bug - the code was in place for making the input always be equal width to the total count, thus preventing the layout shift - see my commit with the fix